web-client/src/views/Public/TrialSessions/PublicTrialSessionDetails.tsx

```diff
@@ -77,16 +77,10 @@ const PublicTrialSessionInformation = ({
               <div>
                 <span className="label">Courthouse location</span>
                 <div className="padding-05"></div>
-                <span>{formattedTrialSession.courthouseName}</span>
-                <span className="address-line">
-                  {formattedTrialSession.address1}
-                </span>
-                <span className="address-line">
-                  {formattedTrialSession.address2}
-                </span>
-                <span className="address-line">
-                  {formattedTrialSession.formattedCityStateZip}
-                </span>
+                <div>{formattedTrialSession.courthouseName}</div>
+                <div>{formattedTrialSession.address1}</div>
+                <div>{formattedTrialSession.address2}</div>
+                <div>{formattedTrialSession.formattedCityStateZip}</div>
```

Comment:
Block display let's these fields wrap at word boundaries rather than in the middle of the text.